overview/templates/overview/candidate_detail.html

```diff
@@ -83,6 +83,24 @@
                 </tr>
             </tr>
             {% endif %}
+
+            {% if endorsements %}
+                <tr>
+                    <th>Endorsements</th>
+                    <td>
+                        {% for endorsement in endorsements %}
+                        <a href="{{ endorsement.link }}" target="_blank" data-toggle="tooltip" title="{{ endorsement.organization.name }} endorsed {{ object.shortname }} on {{ endorsement.date|date }}">
```

Comment:
date isn't required (and I think that's good because we won't always have that data), but this isn't a conditional so when there's no date it'll say "blahblah endorsed on"